REVIEWING.md

```diff
@@ -0,0 +1,66 @@
+
+- [Zelda 64 Reverse Engineering Website](https://zelda64.dev/) - Our homepage, with links to other resources.
+- [Introduction to OOT decomp](https://github.com/zeldaret/oot/blob/master/docs/tutorial/contents.md) - A very detailed tutorial on how to get started with decomp. For OOT, but largely applicable to MM as well.
+- The `#resources` channel on the Discord contains many more links
+
+Getting Started
+---------------
+
+### What should I know to take part in the review process?
+
+You should first get yourself familiar with our [Contributing guide](https://github.com/zeldaret/mm/blob/master/CONTRIBUTING.md). It is also recommended that you have already successfully submitted a merged pull request to understand how the process works before submitting a review.
+
+Pull Requests (PRs)
+------------
+
+### How to review a Pull Request
+
+- When reviewing a PR it is suggested that you follow the checklist outlined in this section.
+- You are not required to check for all of the items in the checklist. It is meant as a guide.
+- Once the PR author has addressed all of your comments, you should add a comment with approval to the PR to signify to the project leads that this PR has been through peer review.
+- If someone does not address your comments and expresses that a different way is better than yours, look for feedback from other contributors. The project lead will have final say in these situations. All decisions will generally be guided by a consensus of contributors.
+
+### Reviewer Checklist
+- [ ] Jenkins build is successful.
+   - [ ] `make` builds a matching ROM.
+   - [ ] Any new compiler warnings that were added are required for matching. Ensure there is good reason if the warnings files have changed. `./tools/warnings_count/warnings_build_current.txt` and `./tools/warnings_count/warnings_setup_current.txt`
+- [ ] Files with NON_MATCHING functions have equivalent behaviour.
+- [ ] `code` and `boot` segment files should be documented as much as possible. Overlays with documentation should be complete.
+   - [ ] Overlays with documentation are required to have macros to define access to parameters if the parameter uses bitwise access. The params should have an enum otherwise.
+- [ ] `format.sh` was run.
+- [ ] Comments and variables have correct spelling.
+- [ ] The following should be declared in an Actor header file. There should be nothing else in the Actor header file.
+   - [ ] Main Actor struct
+   - [ ] Extern'd initVar data.
+   - [ ] Types used in the actor struct. Specific example would be actionFunc typedefs.
+   - [ ] Param field macros and/or enums.
+   - [ ] We have proof that enum/struct/define is needed in another file.
+- [ ] New variables and functions should follow standard naming conventions.
+   - [ ] Constants are converted to whichever looks best in context: hexadecimal, decimal, or float
+	  - [ ] Rotation angles should always be in hexadecimal
```

Comment:
I think something similar to this is that we wanted colors in decimal (that might have been an OoT thing? can't remember)

Reply:
OoT does colours in decimal. There's lots of examples that suggest the devs liked round decimal numbers for colours.

Reply:
So far in Discord the consensus seems to be decimal.